src/main/java/seedu/duke/data/exercisegenerator/GenerateExercise.java

```diff
@@ -190,6 +191,17 @@ public ArrayList<ExerciseData> generateFilteredWorkoutTypeFrom (ArrayList<Exerci
         }
         return filteredExerciseList;
     }
+    //@@ghzr0
+    public ArrayList<ExerciseData> generateIPPTExercises (ArrayList<ExerciseData> exerciseList){
+        assert exerciseList != null : "Exercise List should not be null.";
+        ipptExercises = new ArrayList<>();
+        for (ExerciseData exercise : exerciseList) {
+            if (exercise.getId().equals("563")  || exercise.getId().equals("686") || exercise.getId().equals("819")) {
+                ipptExercises.add(exercise);
```

Comment:
This might cause testers to report a feature flaw as exercise 819 is not a 2.4 run
Maybe can add another exercise in the resources json file

Furthermore, if you want to access id ```123``` you can just use .get(123), better run time 👍